pkg/task_def.go

```diff
@@ -19,6 +19,7 @@ type TaskDef struct {
 	Steps       []Step       `yaml:"steps,omitempty"`
 	Autoenv     bool         `yaml:"autoenv,omitempty"`
 	Autodir     bool         `yaml:"autodir,omitempty"`
+	BindEnvVars bool         `yaml:"bind_env_vars,omitempty"`
```

Comment:
Can we later change this to `bindParamsFromEnv`, so that it is clearer where the envvars are bound to?

Also I prefer lowerCamelCase for config keys in variant :)

Reply:
@mumoshu Thank you for your suggestions, these names are much better indeed 👍. Fixed in https://github.com/mumoshu/variant/pull/107